tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -2246,6 +2246,12 @@ static int prepareForDump(const char *Main,
   InitInvok.setMainExecutablePath(fs::getMainExecutable(Main,
     reinterpret_cast<void *>(&anchorForGetMainExecutable)));
   InitInvok.setModuleName("swift_ide_test");
+#if __APPLE__
+  InitInvok.getLangOptions().EnableObjCInterop = true;
+#else
+  // Ensure the tool works on linux properly
+  InitInvok.getLangOptions().EnableObjCInterop = false;
+#endif
```

Comment:
:-( This should really depend on the target triple, the way it does in the driver.

Reply:
Fixed, mind taking another look?

Reply:
Thank you!